phpBB/phpbb/db/migration/data/v32x/enable_plupload_config.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v32x;
+
+class enable_plupload_config extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\v329',
+		);
+	}
+
+	public function update_data()
+	{
+		return array(
+			array('config.add', array('img_quality', '90')),
+			array('config.add', array('img_strip_metadata', '1')),
```

Comment:
Why true as per default? Current behaviour is does not strip them.

Reply:
Metadata should be stripped by default. The current behaviour is wrong, as users and board administrators have not consented to sensitive metadata being uploaded, nor are they notified it is taking place.  This should be an opt-in feature, as preserving metadata is likely to be required for exceptional cases only.  Weighing up the value of perserving metadata vs the risk of inadvertent disclosure and misuse by bad actors, stripping metadata is the better option.  This is why most platforms that allow image uploading strip metadata by default. Boards that require it can enable the feature.

Reply:
Usually the old behaviour is kept though.

Reply:
Yes normally it would make sense to preserve old default behaviour, however here the motivator for the PR was to change the old behaviour, as discussed in https://www.phpbb.com/community/viewtopic.php?f=436&t=2528276